devel/mxml/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        michaelrsweet mxml 3.1 v
+
+categories          devel textproc
+platforms           darwin
+license             Apache-2.0
+maintainers         nomaintainer
+
+description         Mini-XML is a tiny XML library that you can use to read and write \
+                    XML and XML-like data files in your application without requiring \
+                    large non-standard libraries.
+long_description    ${description} Mini-XML only requires an ANSI C compatible compiler \
+                    (GCC works, as do most vendors’ ANSI C compilers) and a make program. \
+                    \n\nThe Mini-XML library is licensed under the Apache License Version 2.0 \
+                    with an exception to allow linking against GPL2/LGPL2-only software.\n
+
+homepage            https://www.msweet.org/mxml/
+
+checksums           rmd160  9dcdb46e427ce5f57fd23fbfec55fc7ad9efe8d3 \
+                    sha256  0ad3cf45a6d4a100be6a743887bba9eca58266849e996b5cd83f2cd525295bb5 \
+                    size    9267294
+
+destroot.args       DSTROOT=${destroot}
+
+# Test on <= 10.6 first
+universal_variant   no
```

Comment:
If it builds universal for i386 x86_64, and there's no obvious endian or bitness checking happening in the configure script, I'd leave the universal variant enabled.

If there is endian or bitness checking, using the muniversal portgroup could help.